src/logger.js

```diff
@@ -63,11 +63,9 @@ class Logger {
 
   emit(type: string, args: LoggingArgs): void {
     const fn = console[type];
-    if (fn == null) {
-      throw new Error(`Unknown \`console\` function or unavailable: ${type}`);
+    if (fn != null) {
+      fn.apply(console, this.prepend(args, 'html-highlighter'));
```

Comment:
@miguel-guedes were you able to test this on Chrome 50? From my experimentation in the console I don't think that this will work, since the issue seemed to be assigning the console logger object to a variable.

Reply:
Yes.  Try:

```js
const type = 'log'; const fn = console[type]; fn.apply(console, ['foo']);
```

Reply:
Also, the following works in the latest V8 engine:

```sh
echo "const type = 'log'; const fn = console[type]; fn.apply(console, ['foo']);" | nodejs
```

Reply:
I missed in the diff here that the target of "apply" has been changed to `console`. This looks good!